source/_components/sensor.uscis.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: USCIS Sensor
+description: "Instructions on how to set up USCIS within Home Assistant."
+date: 2018-03-21 08:00
+sidebar: false
+comments: false
+sharing: true
+footer: true
+logo: uscis.jpg
+ha_category: Sensor
+ha_release: 0.68
+ha_iot_class: "Cloud Polling"
+---
+
+The `uscis` sensor component allows you get updates on you USCIS case using your case number/ receipt number
+
+## Example for `configuration.yaml` :
+
+```yaml
+sensor:
+  - platform: uscis
+    case_id: REQUIRED_CASE_NUMBER
```

Comment:
Is this a number or a string (as explained below)? IIRC the example should be a real working example one can simply copy & paste to his/her config with minor changes.